xbus-503-ipython-demos/readme.md

```diff
@@ -1,14 +1,14 @@
 # Overview
 
-This project contains demonstration code for the XBUS-503 Data Wrangling course at Georgetown University CCPE.  All code makes use of [iPython Notebook](http://ipython.org/notebook.html) which allows one to interactively run Python code within a web browser.
+This repository contains demonstration code for the XBUS-503 Data Wrangling course at Georgetown University CCPE. The code should be run in a [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/), which allows one to interactively run Python code within a web browser.
 
 ## BeautifulSoup
 
-This notebook demonstrates the ease with which one can use [BeautifulSoup](http://www.crummy.com/software/BeautifulSoup/) to efficiently navigate and parse HTML.  
+This notebook demonstrates the ease with which one can use [BeautifulSoup](http://www.crummy.com/software/BeautifulSoup/bs4/doc/) to efficiently navigate and parse HTML.  
 
 ## Pandas
 
-This notebook (originally authored by Abhijit Dasgupta) demonstrates the use of [pandas](http://pandas.pydata.org/) to quickly analyze the [MovieLens](http://grouplens.org/datasets/movielens/) database.
```

Comment:
The author is also mentioned at the beginning of the notebook so I removed the reference here.